src/en/space-station-14/mapping/guides/general-guide.md

```diff
@@ -188,12 +191,12 @@ In order to create a life support system aboard your map you'll need to create a
 * **Gas Condenser** - Converts gasses into reagents. Probably more important to chemistry or science than atmospherics.
 
 ### Standard Pipe Colors
-Prefer using standard pipe colors for your atmospherics layout so that players and other mappers can understand it better. Consider an in service ship or station won't often have vibrant colors.
+Prefer using standard pipe colors for your atmospherics layout so that players and other mappers can recognize them immediately.
 
-- Waste loop: #990000 (dull red)
-- Distro loop: #0055cc (subdued blue)
-- Air: #03fcd3 (cyan)
-- Mix: #947507 (brownish)
+- Waste loop: #FF1212FF (red)
+- Distro loop: #0335FCFF (blue)
+- Air: #03FCD3FF (cyan)
+- Mix: #947507FF (brownish)
```

Comment:
I don't like that we're changing the standards because mappers can't read the standards, but fine.

Reply:
I've checked multiple maps and everyone is using raw colors instead of the custom distro/waste colors

Reply:
That's not what I said, but it's fine.